R/AffineScoreFunction.R

```diff
@@ -29,7 +29,7 @@ AffineScore <- function(scores, coefs, intercept) {
         stop("intercept must have length 1")
     if (any(!is.finite(c(coefs, intercept))))
         stop("scores and intercept must be finite")
-    new("AffineScore", scores = scores, coefs = coefs, intercept = intercept)
+    new("AffineScore", scores = c(scores), coefs = coefs, intercept = intercept)
```

Comment:
what's happening here? why do we need the c()? where did it break down?

Reply:
When scores just has one element it needs to be converted to a vector.
Returned an error, e.g., here:
https://github.com/kkmann/otsd/blob/69c15f2fafaaa7d0f4210a2ee458eade9ececbc6/tests/testthat/test_AffineScoreFunction.R#L39

Reply:
Cool! And it now works and both cases are tested?

Reply:
Yes as we are adding scores in the minimization 
But I can write a specific test for this!

Reply:
please! makes it easier to identify if we break it again ;)

Reply:
whenever you fix something plz add a test case making sure it's not going to happen again.

Reply:
Alright! I also add tests to achieve 100% coverage